java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -1543,4 +1544,11 @@ public CompletableFuture<Map<String, String[]>> blmpop(
                 arguments,
                 response -> castMapOfArrays(handleMapOrNullResponse(response), String.class));
     }
+
+    @Override
+    public CompletableFuture<String> lset(
+            @NonNull String key, @NonNull long index, @NonNull String element) {
```

Comment:
nonnull for `long` is an overkill though

Reply:
true